src/srb2kart/serverinfo/serverinfo.go

```diff
@@ -0,0 +1,297 @@
+package serverinfo

```

Comment:
Here's how I envisioned the package structure:
GoBun/srb2kart/network
  Contains all the basic functions, these pass the call towards the package specific parsers
GoBun/srb2kart/network/serverinfo
  Contains the content of this file

Reply:
So would `GoBun/srb2kart/network` effectively contain the public parts of the current `serverinfo` package and `GoBun/srb2kart/network` would contain the private parts?

Reply:
I'd say that the body structs can be part of `GoBun/srb2kart/network/info` (there might be a better name) while the public function, `GetSRB2Info`, is public in `GoBun/srb2kart/network`